content/03.categorize.md

```diff
@@ -108,16 +98,16 @@ In other domains, this has resulted in a minimal change in the estimated accurac
 
 Rich clinical information is stored in EHRs.
 However, manually annotating a large set requires experts and is time consuming.
-For chest X-ray studies, a radiologist usually spends a few minutes per example. 
+For chest X-ray studies, a radiologist usually spends a few minutes per example.
 Generating the number of examples needed for deep learning is infeasibly expensive.
 Instead, researchers may benefit from using text mining to generate annotations [@doi:10.3115/1572392.1572411], even if those annotations are of modest accuracy.
 Wang et al. [@arxiv:1705.02315] proposed to build predictive DL models through the use of images with *weak labels*.
 Such labels are automatically generated and not verified by humans, so they may be noisy or incomplete.
-In this case, they applied a series of natural language processing (NLP) techniques to the associated chest X-ray radiological reports. 
-They first extracted all diseases mentioned in the reports using a state-of-the-art NLP tool, then applied a newly-developed negation and uncertainty detection tool (NegBio [@arxiv:1712.05898]) to filter negative and equivocal findings in the reports. 
+In this case, they applied a series of natural language processing (NLP) techniques to the associated chest X-ray radiological reports.
```

Comment:
Check NLP not previously defined

Reply:
I confirmed this is the first use of NLP.  @cgreene didn't edit this section, it shows up in the diff because trailing spaces were removed.